.aws/first.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+PASSWORD=1234567890
+ACCOUNT=main_account
+PORT=26657
+API_PORT=8080
+NETWORK=testnet
+AMOUNT=100stake
+
+./gaiad init --home . --moniker ${ACCOUNT} --chain-id ${NETWORK}
```

Comment:
Thos steps could need some comments

Reply:
Added some line by line / every two lines for coupled commands